search-app/src/cmr/search/models/query.clj

```diff
@@ -215,6 +227,16 @@
    :echo-compatible? false
    :all-revisions? false})
 
+(defmethod common-qm/concept-type->default-query-attribs :autocomplete
+  [_]
+  {:condition (common-qm/->MatchAllCondition)
+   :page-size :unlimited
```

Comment:
Should not be unlimited. Should be `default-page-size` from `cmr.common-app.services.search.query-model`